chrome/content/zotero/xpcom/translation/translate_item.js

```diff
@@ -786,6 +786,19 @@ Zotero.Translate.ItemGetter.prototype = {
 		this.numItems = this._itemsLeft.length;
 	},
 	
+	"setGroup":function(group, getChildCollections) {
+		this._itemsLeft = Zotero.DB.columnQuery('select itemID from items where libraryID in (select libraryID from groups where groupID = ?) and not itemID in (select itemID from deletedItems)', [group.id]);
```

Comment:
This will all need to be redone to not be group-specific, but there shouldn't be any SQL code in here. It should call Zotero.Items.getAll(), as setAll() does below, but with a libraryID for the second parameter. (I'm fixing the messy argument order in 5.0.)